src/components/Avatar/index.tsx

```diff
@@ -1,11 +1,12 @@
 import styled from '@emotion/styled'
 import PropTypes from 'prop-types'
```

Comment:
migrate to Ts & keep proptypes ?

Maybe https://openbase.com/js/babel-plugin-typescript-to-proptypes could help DRY ;)

Reply:
You're right, we've decided to keep them both while migrating but an automated way to convert types into proptypes would be of great value when completed :)